lua/catppuccin/groups/integrations/dap_ui.lua

```diff
@@ -24,13 +24,21 @@ function M.get()
 		DapUIBreakpointsDisabledLine = { fg = C.surface2 },
 
 		DapUIStepOver = { fg = C.blue },
+		DapUIStepOverNC = { fg = C.blue },
```

Comment:
Hey, thanks for the PR!
Out of curiosity, are we unable to use the `link` function like above with `DapUIFloatBorder` to link the `NC` highlights to the normal ones? 

Something like:

```suggestion
		DapUIStepOverNC = { link = "DapUIStepOver" },
```

I would appreciate if you could let me know if that works or not as I currently can't try it out myself.

Reply:
That should work! I pushed a different commit :)